packages/style-build/README.md

```diff
@@ -0,0 +1,18 @@
+# Style Build Helper
+
+This is a [Webpack](https://webpack.js.org/) config generator for building WooCommece component styles using base styles from Gutenberg. It replaces the [`bin/packages/build.js`](https://github.com/woocommerce/woocommerce-admin/blob/6859249/bin/packages/build.js) script.
+
+
+## Usage
+
+***Note:*** Your package's main style entry must be `src/style.scss`.
```

Comment:
If each package must declare its `entry` property, this wouldn't be exactly true, unless we'd like to extract this to `@woocommerce/style-build`

Reply:
Nice catch - I forgot to clean this up after b337e27.